src/desktop.md

```diff
@@ -5,32 +5,31 @@ toc: true
 ---
 
 Desktop support allows you to compile Flutter source code
-to a native macOS or Linux desktop app. Flutter's desktop
+to a native Windows, macOS, or Linux desktop app. Flutter's desktop
 support also extends to plugins&mdash;you can install 
 existing plugins that support the macOS or Linux platforms,
 or you can create your own.
 
 {{site.alert.warning}}
   **Work in progress!**
-  This page covers desktop support for macOS and Linux,
-  which are available as alpha-quality features in the Flutter dev channel.
-  Windows platform support is still under development.
-
-  These platforms still have notable feature gaps,
+  This page covers desktop support,
+  which is available as alpha-quality
+  features in the Flutter dev channel.
+  Support still has notable feature gaps,
   including accessibility support.
-  We strongly recommend that you examine the
-  [Desktop shells][] page in the [Flutter wiki][]
-  to understand known limitations and ongoing work.
 {{site.alert.end}}
 
 {{site.alert.note}}
-  To compile a macOS desktop app, you must build the app on a Mac.
-  To compile a Linux desktop app, you must build the app on Linux.
+  To compile a desktop app, you must build the app **on**
+  the targeted platform: build a Windows app on Windows,
+  a macOS app on macOS, and so on.
```

Comment:
there's only one more platform left, so perhaps "and so on" becomes "and a Linux app on Linux"